torchvision/models/detection/generalized_rcnn.py

```diff
@@ -57,6 +57,16 @@ def forward(self, images, targets=None):
         """
         if self.training and targets is None:
             raise ValueError("In training mode, targets should be passed")
+        if self.training:
+            boxes = targets["boxes"]
```

Comment:
`targets` is a list of `dict`, so you might need to iterate over it checking that each element in the list is of the correct shape.
Something in the lines of
```python
assert targets is not None
for target in targets:
    boxes = target["boxes"]
    ...
```
Also, you might need to assert that the `targets` is not None, otherwise torchscript might not be able to infer it.

Reply:
Done!